src/apis/attachments/dto/attachment.dto.ts

```diff
@@ -0,0 +1,48 @@
+import { ApiProperty } from '@nestjs/swagger';
+
+import { Attachment } from '@src/entities/Attachment';
+
+export class AttachmentDto implements Omit<Attachment, 'user'> {
+  @ApiProperty({
+    description: '첨부 파일 고유 ID',
+    format: 'integer',
+  })
+  id: string;
+
+  @ApiProperty({
+    description: '업로더 고유 ID',
+    format: 'integer',
+  })
+  userId: number;
+
+  @ApiProperty({
+    description: 'file url',
+  })
+  url: string;
+
+  @ApiProperty({
+    description: 'domain을 제외한 path',
+  })
+  path: string;
+
+  @ApiProperty({
+    description: 'MIME-Type',
+  })
+  mimeType: string;
+
+  @ApiProperty({
+    description: '파일 용량(byte)',
+    format: 'integer',
+  })
+  capacity: string;
```

Comment:
```suggestion
  @ApiProperty({
    type: 'number',
    description: '파일 용량(byte)',
    format: 'int64',
  })
  capacity: string;
```
이렇게 돼야할거같아요 capacity는 서버 내부에서 값을 어떻게 관리하냐에 따라서 타입 지정해주시면 될거같고요
https://swagger.io/docs/specification/data-models/data-types/